examples/script/script.ts

```diff
@@ -0,0 +1,24 @@
+import { getContext } from '@keystone-6/core/system';
+import config from './keystone';
+import * as PrismaModule from '.prisma/client';
+
+async function main() {
+  const { connect, context, disconnect } = getContext(config, PrismaModule);
+
+  console.log('(script.ts)', 'connect');
+
+  // if you don't call connect, db.onConnect will not be run
+  // but Prisma will automatically connect when Prisma is used
```

Comment:
```suggestion
  // connect() here also runs db.onConnect from your keystone config, 
  // to avoid running db.onConnect you can not run connect here, 
  // in which case Prisma will automatically connect when it is used
```